terraform/api.tf

```diff
@@ -0,0 +1,142 @@
+#
+# Security Group Resources
+#
+resource "aws_security_group" "alb" {
+  name   = "sg${var.project}APILoadBalancer"
+  vpc_id = data.terraform_remote_state.core.outputs.vpc_id
+
+  tags = {
+    Name    = "sg${var.project}APILoadBalancer",
+    Project = var.project
+  }
+}
+
+#
+# ALB Resources
+#
+resource "aws_lb" "api" {
+  name            = "alb${var.project}API"
+  security_groups = [aws_security_group.alb.id]
+  subnets         = data.terraform_remote_state.core.outputs.public_subnet_ids
+
+  enable_http2 = true
+
+  tags = {
+    Name    = "alb${var.project}API"
+    Project = var.project
+  }
+}
+
+resource "aws_lb_target_group" "api" {
+  name = "tg${var.project}API"
+
+  health_check {
+    healthy_threshold   = 3
+    interval            = 30
+    matcher             = 200
+    protocol            = "HTTP"
+    timeout             = 3
+    path                = "/"
```

Comment:
I believe that there was talk about using the OpenAPI YAML as the health check endpoint. Should that be used here vs. `/`?

Reply:
Yeah, we can use that. I used `/` and heard back about the other endpoint being available after the PR was up.